rbac/management/workspace/utils.py

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+from uuid import UUID
+from rest_framework.serializers import ValidationError
+
+
+from management.role.model import Access
+from management.utils import roles_for_principal
+from management.workspace.model import Workspace
+
+
+def is_user_allowed(request, required_operation, target_workspace):
+    """Check if the user is allowed to perform the required permission on the target workspace."""
+    default_workspace_id = str(Workspace.objects.get(request.tenant, type=Workspace.Types.DEFAULT).id)
```

Comment:
We should be able to use the manager here? https://github.com/RedHatInsights/insights-rbac/blob/89efd830ee143e7bfb4460e59136cccab13f2954/rbac/management/managers.py#L55

```suggestion
    default_workspace_id = str(Workspace.objects.default(tenant_id=request.tenant).id)
```